pkg/convert/workflow.go

```diff
@@ -0,0 +1,129 @@
+package convert
+
+import (
+	"fmt"
+	"sort"
+
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/tinkerbell/tink/pkg/apis/core/v1alpha1"
+	protoworkflow "github.com/tinkerbell/tink/protos/workflow"
+	"github.com/tinkerbell/tink/workflow"
+)
+
+func WorkflowYamlToCRD(wf *workflow.Workflow, crd *v1alpha1.Workflow) {
```

Comment:
```suggestion
func WorkflowYAMLToCRD(wf *workflow.Workflow, crd *v1alpha1.Workflow) {
```
So all the acronyms are in all caps please.